pycircstat/tests.py

```diff
@@ -234,3 +235,110 @@ def _critical_value_raospacing(n, U):
         p = .5
 
     return p, Uc
+    
+    
+def vtest(alpha, mu, w=None, d=None, axis=0):
```

Comment:
Didn't we say that axis=None by default? I implemented everything else like that. I think this would be better since it mimics the usual numpy behaviour. So axis=None -> everything is flattened first, axis=somethingelse -> test computed along something else. I wrote a decorator to switch everything to axis=0 if axis is not None. Therefore, you could just leave most of the code and adapt it.